virtool/tasks/runner.py

```diff
@@ -53,4 +71,9 @@ async def run_task(self, task_id: int):
         for cls in BaseTask.__subclasses__():
             if task.type == cls.name:
                 current_task = await cls.from_task_id(self.app["data"], task.id)
-                await asyncio.get_event_loop().create_task(current_task.run())
+
+                self.current_task = asyncio.get_event_loop().create_task(
```

Comment:
Please check if this works. I don't think it is necessary to get the event loop.
```suggestion
                self.current_task = asyncio.get_event_loop().create_task(
```